manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml

```diff
@@ -53,3 +53,46 @@ spec:
   selector:
     matchLabels:
       app: oauth-openshift
+---
```

Comment:
I don't have enough expertise to review this file but I trust you with what you are doing.

Reply:
A comment explaining why specific labels are dropped will really help a person like me who is  unfamiliar with these metrics.

Reply:
Honestly, I am just as unfamiliar with the metrics as you are. This is copied from the openshift-apiserver operator.